internal/cloud/atlas/client.go

```diff
@@ -49,8 +50,16 @@ type client struct {
 	transport *digest.Transport
 }
 
-func (c *client) do(method, path string, options api.RequestOptions) (*http.Response, error) {
-	req, reqErr := http.NewRequest(method, c.baseURL+path, options.Body)
+func (c *client) doURL(method, path string, options api.RequestOptions) (*http.Response, error) {
```

Comment:
I'm not sure this extra private method needs to exist if its solely passing through to `c.do`...